### PR TITLE
Use scoped Prometheus Registry instead of global

### PR DESCRIPTION
**Pull request by metalmatze**
/cc @squat

### CODE REVIEW

File: pkg/http/server.go [hunk 1]
```diff
@@ -3,46 +3,58 @@ package http
...
 
-func NewInstrumentedHandler(handlerName string, next http.Handler) http.Handler {
-	return promhttp.InstrumentHandlerDuration(
-		requestDuration.MustCurryWith(prometheus.Labels{"handler": handlerName}),
-		promhttp.InstrumentHandlerRequestSize(
-			requestSize.MustCurryWith(prometheus.Labels{"handler": handlerName}),
-			promhttp.InstrumentHandlerCounter(
-				requestsTotal.MustCurryWith(prometheus.Labels{"handler": handlerName}),
+func (i InstrumentHandler) Handle(handlerName string, next http.Handler) http.Handler {
+	labels := prometheus.Labels{"handler": handlerName}
```
Comment (squat) [inline pkg/http/server.go]: :sunglasses:

**Comment by metalmatze** [comment]
We actually don't care too much about this compared to the time we would need to improve this...

**Pull request closed without merging**